packages/react/select/src/Select.stories.tsx

```diff
@@ -430,7 +430,104 @@ export const WithinDialog = () => (
   </Dialog.Root>
 );
 
-export const ChromaticShortOptions = () => (
+export const ChromaticShortOptionsPaddedContent = () => (
+  <ChromaticStoryShortOptions padding="content" />
+);
+ChromaticShortOptionsPaddedContent.parameters = { chromatic: { disable: false } };
+
+export const ChromaticShortOptionsPaddedViewport = () => (
+  <ChromaticStoryShortOptions padding="viewport" />
+);
+ChromaticShortOptionsPaddedViewport.parameters = { chromatic: { disable: false } };
+
+export const ChromaticLongOptionsPaddedContent = () => (
+  <ChromaticStoryLongOptions padding="content" />
+);
+ChromaticLongOptionsPaddedContent.parameters = { chromatic: { disable: false } };
+
+export const ChromaticLongOptionsPaddedViewport = () => (
+  <ChromaticStoryLongOptions padding="viewport" />
+);
+ChromaticLongOptionsPaddedViewport.parameters = { chromatic: { disable: false } };
+
+export const ChromaticTopFirstPaddedContent = () => <ChromaticStoryTopFirst padding="content" />;
+ChromaticTopFirstPaddedContent.parameters = { chromatic: { disable: false } };
+
+export const ChromaticTopFirstPaddedViewport = () => <ChromaticStoryTopFirst padding="viewport" />;
+ChromaticTopFirstPaddedViewport.parameters = { chromatic: { disable: false } };
+
+export const ChromaticBottomLastPaddedContent = () => (
+  <ChromaticStoryBottomLast padding="content" />
+);
+ChromaticBottomLastPaddedContent.parameters = { chromatic: { disable: false } };
+
+export const ChromaticBottomLastPaddedViewport = () => (
+  <ChromaticStoryBottomLast padding="viewport" />
+);
+ChromaticBottomLastPaddedViewport.parameters = { chromatic: { disable: false } };
+
+type PaddedElement = 'content' | 'viewport';
+
+interface ChromaticSelectProps extends React.ComponentProps<typeof Select.Trigger> {
+  count?: number;
+  padding?: PaddedElement;
```

Comment:
Would it be clearer to name the prop `paddedElement` as well? with `padding` as a prop name I instinctively expect it to directly accept a style value for some reason.